api/rest/v1/query.md

```diff
@@ -0,0 +1,189 @@
+---
+layout: default
+title: SPARQL
+nav_order: 12
+parent: REST (v1)
+grand_parent: API
+published: false
+permalink: /api/rest/v1/query
+---
+
+# /v1/query
+
+Query the Data Commons knowledge graph using
+[SPARQL](https://www.w3.org/TR/rdf-sparql-query/).
+
+This endpoint makes it possible to query the Data Commons knowledge graph using
+SPARQL. SPARQL is a query language developed to retrieve data from websites. It
```

Comment:
I am not sure if it's for websites, can we borrow something from https://www.w3.org/TR/sparql11-query/

Reply:
Changed the description from 'websites' to 'RDF graph content on the web" for both v1 and v0 pages.